provisioner/plain-v0/README.md

```diff
@@ -1,3 +1,197 @@
-# plain-v0
+# Plain-v0 Provisioner
 
-A concrete Provisioner implementation for "plain" k8s bundles.
+## Summary
+
+The `plain-v0` provisioner is a core rukpak provisioner that knows how to interact with `plain+v0` format bundles.
+The `plain+v0` bundles, or plain bundles, are simply container images containing a set of static kubernetes YAML
+manifests in a given directory. For example, below is an example Dockerfile that builds a plain `plain+v0` bundle from a
+directory containing static manifests.
+
+```dockerfile
+FROM scratch
+LABEL core.rukpak.io.bundle.mediatype.v1=plain+v0
+LABEL core.rukpak.io.bundle.manifests.v1=/manifests
+LABEL core.rukpak.io.bundle.package.v1=my-package
+LABEL core.rukpak.io.bundle.version.v1=0.0.1
+COPY ./manifests /manifests
```

Comment:
can we add a note that we explicitly need the directory of manifests to live in the /manifests for the provisioner to work?